tests/plugins/tableselection/tableselection.js

```diff
@@ -145,6 +145,30 @@
 			mockMouseSelection( editor, [ cells.getItem( 1 ), cells.getItem( 3 ), cells.getItem( 8 ) ], function() {
 				assert.pass();
 			} );
+		},
+
+		// (#1887)
+		'test toggling class while focusing/bluring editor': function( editor ) {
```

Comment:
For this ticket I'd go with implementation-agnostic unit test - based on (computed) color of a cell's background.